hadoop-project/pom.xml

```diff
@@ -167,9 +167,11 @@
     -->
     <enforced.java.version>[${javac.version},)</enforced.java.version>
     <enforced.maven.version>[3.3.0,)</enforced.maven.version>
-
+    <extraJavaTestArgs>
```

Comment:
jdk8 does not support --add-exports, and the test will fail.

Reply:
read [here](https://github.com/apache/hadoop/pull/6939#discussion_r1677448627)